kafka-connect-aws-s3/src/test/scala/io/lenses/streamreactor/connect/aws/s3/sink/config/DeleteModeSettingTest.scala

```diff
@@ -0,0 +1,27 @@
+package io.lenses.streamreactor.connect.aws.s3.sink.config
+
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.prop.TableDrivenPropertyChecks._
+
+import scala.jdk.CollectionConverters.MapHasAsJava
+
+class DeleteModeSettingTest extends AnyFlatSpec with Matchers with LazyLogging {
```

Comment:
To match the name of the class it's testing, this should be `DeleteModeSettingsTest`